src/signals/incident-management/containers/IncidentSplitContainer/IncidentSplitFormIncident.js

```diff
@@ -0,0 +1,106 @@
+import React, { useCallback, useState, Fragment } from 'react';
```

Comment:
This is also a component and can be placed in the `components` subfolder.
The filename is a bit strange. Wouldn't be `IncidentSplitPart` more appropriate?

Reply:
I would prefer to move the whole `container` to `/components/IncidentSplit` and rename components accordingly.

Then you would get something like: `SplitFormIncident`

About the `part` thing, IMHO this is not descriptive, `IncidentSplitFormIncident` is in the current context part of `IncidentSplitForm` and does not show a `part` but a 'splitted' incident, like the name describes.